packages/react/src/components/SpinButton/SpinButton.styles.ts

```diff
@@ -192,49 +191,34 @@ export const getStyles = (props: ISpinButtonStyleProps): ISpinButtonStyles => {
         boxSizing: 'border-box',
         height: DEFAULT_HEIGHT,
         minWidth: DEFAULT_MIN_WIDTH,
-        selectors: {
-          // setting border using pseudo-element here in order to prevent:
-          // input and chevron buttons to overlap border under certain resolutions
-          ':after': {
-            pointerEvents: 'none',
-            content: "''",
-            position: 'absolute',
-            left: 0,
-            top: 0,
-            bottom: 0,
-            right: 0,
-            borderWidth: '1px',
-            borderStyle: 'solid',
-            borderColor: SpinButtonRootBorderColor,
-            borderRadius: effects.roundedCorner2,
+        ...getInputFocusStyle(SpinButtonRootBorderColor, effects.roundedCorner2),
+        ':after': {
+          borderWidth: '1px',
+          [HighContrastSelector]: {
```

Comment:
The getInputFocusStyle method uses a selectors property in between after and the HighContrastSelector.  Is that needed here too?

Reply:
the idea here is that spin button sets the base outline using getInputFocusStyle, and then changes the color/thickness for hover and focused state. 

I agree it's not great to use 'focusStyle' for a non focus style state, but its better than simply recreating the styles manualy